types/react-bootstrap-table-next/index.d.ts

```diff
@@ -378,15 +378,15 @@ export interface SizePerPageRendererOptions {
     /**
      * dropdown options
      */
-    options: Array<{ text: string; value: number }>;
+    options: Array<{ text: string; page: number }>;
     /**
      * current size per page
      */
-    currentSizePerPage: number;
+    currSizePerPage: string;
     /**
      * call it when you need to change size per page
      */
-    onSizePerPageChange: (page: number, sizePerPage: number) => void;
+    onSizePerPageChange: (page: number) => void;
```

Comment:
Reason:
See https://github.com/Avasam/speedrun.com_global_scoreboard_webapp/blob/develop/global-scoreboard/src/Components/Dashboard/TableElements/PaginationProps.tsx#L5 for usage

Reference from source code:
https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/master/packages/react-bootstrap-table2-paginator/src/size-per-page-dropdown-adapter.js#L45
https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/master/packages/react-bootstrap-table2-paginator/src/page-resolver.js#L162